src/main/java/util/Utility.java

```diff
@@ -72,7 +72,10 @@ public static boolean writeFile(String fileName, String content, int issueCount)
                 writer.println(content);
                 writer.close();
 
-                long sizeAfterWrite = Files.size(Paths.get(fileName));
+                long sizeAfterWrite = 0;
+                if (issueCount > 0) {
```

Comment:
Is this check necessary if you only carry out the check in `processFileGrowth` when `issueCount` is nonzero?